supplier_app/models.py

```diff
@@ -73,6 +84,11 @@ class TaxPayerArgentina(TaxPayer):
         verbose_name='Certificate of no tax withholding of IVA, income or SUSS',
         validators=[FileExtensionValidator(allowed_extensions=['pdf'])],
         )
+    comments = models.CharField(
+        max_length=200,
+        blank=True,
+        null=True
+        )
```

Comment:
`comments` field is only available for `TaxPayerArgentina` or should be generic?

Reply:
Resolved, comments is now available for every taxpayer